File: Quest/Assets/Src/Cards/Ally Cards/QueenIseult.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class QueenIseult : AllyCard
{

    public QueenIseult() : base("Queen Iseult", 0, 2) //base 10
    {

    }

    public override int getBattlePoints(GameState gState) //override 20
    {
        //Player[] players = gState.getPlayers();
        foreach (Player player in gState.getPlayers())
        {
            if (player.getPlayersBoard().Equals("Sir Tristan"))
            {
                return bidPower = 4;
            }
            return bidPower = 2;
        }
        return bidPower = 2;
    }

}

File: eclipse/Quest/src/model/cards/allyCards/QueenIseult.java
package model.cards.allyCards;

import model.cards.AllyCard;
import model.game.GameState;
import model.player.Player;

public class QueenIseult extends AllyCard{

	//private static final String Player = null;

	public QueenIseult() {
		super("Queen Iseult", 0, 2); //base 10
	}

    public int getBattlePoints(GameState gState) //override 20
    {
        //Player[] players = gState.getPlayers();
        for (Player player : gState.getPlayers())
        {
            if (player.getPlayersBoard().equals("Sir Tristan"))
            {
                return bidPower = 4;
            }
            return bidPower = 2;
        }
        return bidPower = 2;
    }
}
